logbook-logstash/src/main/java/org/zalando/logbook/logstash/LogstashLogbackSink.java

```diff
@@ -27,19 +27,41 @@ public final class LogstashLogbackSink implements Sink {
 
     private final String baseField;
 
+    private final String level;
```

Comment:
Why not use `org.slf4j.event.Level` instead? In that case, we'd be able to use [atLevel](https://www.slf4j.org/api/org/slf4j/Logger.html#atLevel-org.slf4j.event.Level) method and not use a custom enum

Reply:
@kasmarian Thank you so much for your valuable comments, I have considered incorporating `org.slf4j.event.Level` as you recommended, aiming to manage logging levels more effectively.

This customization doesn't alter the default behavior of LogstashLogbackSink but provides  more flexibility for users.

Reply:
Thank you! Looks like the test coverage is lagging behind a bit, if you could fix it as well. Otherwise LGTM

Reply:
@kasmarian 

I have corrected and verified that the coverage is indeed 100%
Thank you !